code/modules/projectiles/projectile/beams.dm

```diff
@@ -55,6 +55,11 @@
 	impact_effect_type = /obj/effect/temp_visual/impact_effect/blue_laser
 	light_color = LIGHT_COLOR_CYAN
 
+/obj/item/projectile/beam/disabler/on_hit(atom/target, blocked, hit_zone)
```

Comment:
This is insane. Don't do this.

Reply:
If the head coders want it removed, I'll do so, but it was implemented for consistency with how tasers previously interacted with the emagged upgraded cybernetic heart.

Remove it, or keep it?